File: 0001 Code Chef/0026 How many unattempted Problem/implementation/0004 java/Unattempted.java
import java.util.Scanner;

public class Unattempted{
    public static void main(String[] args){
        int total, attempted, unattempted;
        String stringInput;
        String[] stringArray;

        Scanner read = new Scanner(System.in);
        stringInput = read.nextLine();
        stringArray = stringInput.split(" ");
        total = Integer.parseInt(stringArray[0]);
        attempted = Integer.parseInt(stringArray[1]);
        unattempted = total - attempted;
        System.out.println(unattempted);
    }
}

File: 0001 Code Chef/0026 How many unattempted Problem/implementation/0011 lua/unattempted.lua
local string_input = io.read();
local string_array = {}
local index = 1
for i in string.gmatch(string_input,"%S+") do
    string_array[index] = i 
    index = index + 1
end

local total = tonumber(string_array[1])
local attempted = tonumber(string_array[2])
local unattempted = total - attempted
print(unattempted)